components/resources/brave_components_resources.grd

```diff
@@ -45,11 +45,16 @@
       </if>
 
       <!-- WebUI wallet resources -->
-      <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page.html" type="BINDATA" />
+      <if expr="is_ios">
+        <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page_ios.html" type="BINDATA" />
+      </if>
+      <if expr="not is_ios">
+        <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page.html" type="BINDATA" />
+      </if>
```

Comment:
Don't recreate the files just to add a few lines, see: https://github.com/brave/brave-core/commit/c6895a85e14ee5976210e289033fc72482f2884f